### PR TITLE
Creation of a single output upgrades file

### PR DESCRIPTION
**Pull request by ann-sherin**
* combines the main upgrade output files into one. 
* runs on latest opendss version 0.7.0 (also related to #151)

### CODE REVIEW

**Review by daniel-thom — APPROVED** [review]

File: disco/cli/upgrade_cost_analysis.py [hunk 1]
```diff
@@ -356,20 +351,31 @@ def aggregate_results(output, fmt, verbose):
...
     }
-
+    empty_tables = {"results": [], 'costs_per_equipment': [], 'violation_summary': [], 'equipment': []}
     for name in job_names:
         job_path = jobs_output_dir / name
         job_info = JobInfo(name)
-        summary_table = get_upgrade_summary_table(job_path, job_info)
-        costs_table = get_total_upgrade_costs_table(job_path, job_info)
+        job_keyword = "name"
+        job_name = getattr(job_info, job_keyword)
```
Comment (daniel-thom) [inline disco/cli/upgrade_cost_analysis.py:367]: Seems like you can just use `name`

File: disco/cli/upgrade_cost_analysis.py [hunk 2]
```diff
@@ -356,20 +351,31 @@ def aggregate_results(output, fmt, verbose):
...
+        overall_output_summary_file = job_path / "overall_output_summary.json"
+        if not overall_output_summary_file.exists():
+           tables = empty_tables
+        try:
+            with open(overall_output_summary_file) as json_file:
+                data = json.load(json_file)
+        except pd.errors.EmptyDataError:
```
Comment (daniel-thom) [inline disco/cli/upgrade_cost_analysis.py:374]: This shouldn't be possible...you're doing a json.load. If there is a parsing error, won't it be fatal?

File: disco/extensions/upgrade_simulation/upgrades/common_functions.py [hunk 3]
```diff
@@ -815,7 +829,10 @@ def get_all_transformer_info_instance(upper_limit=None, compute_loading=True):
     for index, row in all_df.iterrows():
         all_df.at[index, "kVs"] = [float(a) for a in row["kVs"]]
         all_df.at[index, "kVAs"] = [float(a) for a in row["kVAs"]]
-        all_df.at[index, "Xscarray"] = [float(a) for a in row["Xscarray"]]
+        try:
```
Comment (daniel-thom) [inline disco/extensions/upgrade_simulation/upgrades/common_functions.py:832]: If you want to check explicitly you can use this. But, you would may need to use something like the semver package to compare versions. Maybe not if you are just checking for 0.6.
```
In [7]: opendssdirect._version.__version__
Out[7]: '0.6.1'
```

Comment (ann-sherin) [reply disco/extensions/upgrade_simulation/upgrades/common_functions.py:832]: Well I left this in cause it seemed a weird change from the previous version 
new version: "[1,2,3]"
old version: ["1", "2", "3"]

All the other arrays are still in the old version format -- so I just feel like this is an unusual change, which could be reverted. So just leaving it in there.

**Pull request merged**